Trimble.Modus.Components/Controls/TMInput/TMInput.xaml.cs

```diff
@@ -479,4 +479,21 @@ private static void OnEnabledPropertyChanged(BindableObject bindable, object old
 
     #endregion
 
+    #region Internal methods
+    internal void ToggleRightIconState(bool enabledState)
```

Comment:
Add a comment over these method to say these are used in the numeric input control

Reply:
Done